app/components/nav-bar.js

```diff
@@ -10,3 +11,11 @@ export default Component.extend({
     }
   }
 });
+
+$(document).ready(function() {
```

Comment:
No. This is not how you're supposed to do things in Ember. The component may not always be ready by the time the document is. Move this inside didInsertElement method of the component class.